commitlint.config.ts

```diff
@@ -264,12 +280,19 @@ module.exports = {
                     }
 
                     // taken from https://stackoverflow.com/a/66433444/544947 and https://unix.stackexchange.com/a/25208/56844
-                    let recommendedUnixCommand =
-                        'git log --format=%B -n 1 $(git log -1 --pretty=format:"%h") | cat - > log.txt ; fmt -w 1111 -s log.txt > ulog.txt && fmt -w 64 -s ulog.txt > wlog.txt && git commit --amend -F wlog.txt';
+                    function getUnixCommand(fmt_option: string){
+                        fmt_option = fmt_option.trim();
+                        if (fmt_option !== '') { 
+                            fmt_option = ' ' + fmt_option;
```

Comment:
@realmarv why not pass the parameter with a space instead of doing this in the function? the less code we have, the better

Reply:
I thought it might be confusing for someone who is using the function. It's a simple function though

Reply:
but you could still add the space in the command string instead of having this logic